tests/demo_native.rs

```diff
@@ -0,0 +1,249 @@
+use anyhow::{anyhow, Result};
+use async_std::{future, task::sleep};
+use client::SequencerClient;
+use espresso_types::FeeAmount;
+use ethers::prelude::*;
+use futures::future::join_all;
+use reqwest::blocking;
+use std::{
+    fmt,
+    str::FromStr,
+    time::{Duration, Instant},
+};
+use surf_disco::Url;
+
+const L1_PROVIDER_RETRY_INTERVAL: Duration = Duration::from_secs(1);
+// TODO add to .env or derive
+const BUILDER_ADDRESS: &str = "0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f";
```

Comment:
why is this hard coded?

Reply:
It is hard coded in the smoke-test bash script

Reply:
https://github.com/EspressoSystems/espresso-sequencer/issues/2055